### PR TITLE
Add a job to run all MCO test cases in an AWS arm64 fips proxy cluster

### PR DESCRIPTION
**Pull request by sergiordlr**
Add prow jobs to run all MCO test cases in an IPI on AWS arm64 fips cluster with proxy

### CODE REVIEW

**Comment by sergiordlr** [comment]
/hold 
WIP

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-aws-ipi-longduration-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g1-f9 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-g2-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-p1-f28 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-p2-f28 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-p3-f28

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml [hunk 1]
```diff
@@ -86,6 +86,66 @@ resources:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      FEATURE_SET: TechPreviewNoUpgrade
+      FIPS_ENABLED: "true"
+      OCP_ARCH: arm64
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;~hypershift&;~P1&;~P2&
+      TEST_IMPORTANCE: Critical|Medium|Low
```
Comment (ptalgulk01) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:143]: Same here

**Review by ptalgulk01 — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml [hunk 2]
```diff
@@ -86,6 +86,66 @@ resources:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      FEATURE_SET: TechPreviewNoUpgrade
+      FIPS_ENABLED: "true"
+      OCP_ARCH: arm64
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;~hypershift&;P1&
+      TEST_IMPORTANCE: High
```
Comment (ptalgulk01) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:103]: I think here we need to remove the test-importance here as we selecting P1 test-cases

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml [hunk 3]
```diff
@@ -86,6 +86,66 @@ resources:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      FEATURE_SET: TechPreviewNoUpgrade
+      FIPS_ENABLED: "true"
+      OCP_ARCH: arm64
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;~hypershift&;P2&
+      TEST_IMPORTANCE: Critical|Medium|Low
```
Comment (ptalgulk01) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:123]: Same here

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml (hunk 1 above)
Comment (sergiordlr) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml]: Done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml (hunk 2 above)
Comment (sergiordlr) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml]: Done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml (hunk 3 above)
Comment (sergiordlr) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml]: Done

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-p1-f28 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-p2-f28 periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-longrun-mco-proxy-fips-p3-f28

**Pull request merged**